tests/test_pdf.py

```diff
@@ -29,6 +29,38 @@ def test_pdf_inputs(backend):
     )
 
 
+@pytest.mark.fail_mxnet
+def test_pdf_basicapi_tests(backend):
+    source = {
+        "binning": [2, -0.5, 1.5],
+        "bindata": {"data": [55.0], "bkg": [50.0], "bkgerr": [7.0], "sig": [10.0]},
+    }
+    pdf = pyhf.simplemodels.hepdata_like(
+        source['bindata']['sig'], source['bindata']['bkg'], source['bindata']['bkgerr']
+    )
+
+    pars = pdf.config.suggested_init()
+    data = source['bindata']['data'] + pdf.config.auxdata
+
+    tensorlib, _ = backend
+    assert tensorlib.tolist(pdf.pdf(pars, data))
```

Comment:
Instead of just asserting the existence of the call, why not also assert against target values?